docs/use.html

```diff
@@ -0,0 +1,722 @@
+<h3>Hyperlinking</h3>
+<p>To create links within your documentation simply use the <code>NAME</code> of the page or stylesheet that has been documented using the tags <code>@page</code> or <code>@stylesheet</code> and wrap it in <code>[]</code>. For example:</p>
+<p>The following page</p>
+<pre><code>@page accesibility Accesibility
+
+</code></pre>
+<p>Can be hyperlinked like this:</p>
+<pre><code>Referece [accesibility] for more details.
+
+</code></pre>
+<p>For links external to your style guide, use <a href="https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links">markdown</a>.</p>
+<h3>Adding images</h3>
+<p>From time to time you may want to add an image to your style guide. For this use as well <a href="https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#images">markdown</a>.</p>
+<h2>Documenting a Stylesheet</h2>
+<p>To document a stylesheet, open an existing stylesheet or create a new one. You can use .CSS, .Less, or .SCSS extensions. Then, at the top of your document add the <a href="use.html#section=section__page">@stylesheet</a> tag inside of a css comment. As with the page tag, you can add a description right below it. For example:</p>
+<pre><code class="language-css">/**
+ *  @stylesheet buttons.less Buttons
+ *  
+ *  Global style definitions for all button elements. 
+ */
+
+</code></pre>
+<p>This will create a page called <code>buttons.less.html</code> in the directory that you chose to generate your documentation in.</p>
+<h3>Documenting Styles</h3>
+<p>The next step is to use the <a href="use.html#section=section__styles">@styles</a> tag to organize the styles inside of your stylesheet. For example, inside of your stylesheet for buttons, you may have styles for colors, sizes, and variations. You can use the <code>@styles</code> tag to create sections and document them separately. Besides adding structure to your document, doing this will also allow you to hyperlink directly to that section, as well as to show it on the summary of links at the top of the documentation for that stylesheet.</p>
+<p>Here's an example of how this tag can be used:</p>
+<pre><code class="language-css">/**
+ *  @stylesheet buttons.less Buttons
+ *  
+ *  Global style definitions for all button elements. 
+ *
+ */
+
+ /**
+ *  @styles buttons-sizes Button Sizes 1
+ *
+ *  There are 3 button sizes that can be used:
+ */
+
+  btn-small {
+  ...
+  }
+ 
+
+  /**
+ *  @styles buttons-colors Button Colors 0
+ *
+ *  Use the color variations to denote importance of each action type :
+ */
+
+  btn-success {
+  ...
+  }
+
+
+</code></pre>
+<h3>Creating Demos</h3>
+<p>Creating demos allows you to include markup and css in your Living Style Guide to demonstrate your styles in action. For this, create and HTML file inside of your <code>demos</code> directory as it was indicated under the <a href="setup.html#section=section_FileOrganization">File Organization</a> section.</p>
```

Comment:
"create **an** HTML file" (or "a" since technically h isn't a vowel, even though it sounds vowel-y when spoken. Either is correct enough)